### PR TITLE
Representations api

### PR DESCRIPTION
**Pull request by gesinger**
Rebase of https://github.com/videojs/videojs-contrib-dash/pull/66

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,3 +68,29 @@ videojs.Html5DashJS.updateSourceData = function(source) {
...
+
+## Interacting with representations
+
+### Get all of the representations and enable/disable them
+
+To get all of the available representations, call the `representations()` method on the `sourceHandler`. This will return a list of plain objects, each with `width`, `height`, `bandwidth`, and `id` properties, and an `enabled()` method.
+
+```
+var sourceHandler = player.tech({ IWillNotUseThisInPlugins: true }).sourceHandler_;
+var representations = sourceHandler.representations();
```
Comment (forbesjo) [inline README.md]: Isn't this `player.dash.representations()` now?

Comment (gesinger) [inline README.md]: You're right. I must've messed that up in the rebase somewhere.

**Comment by forbesjo** [comment]
Closed because we would rather add this to dash.js itself and keep this plugin as simple as possible.

**Comment by git03394538** [comment]
@forbesjo I am trying to implement quality-picker too. You have mentioned that you rather add this feature to dash.js, but I couldn't find it. Can you point me to the right direction?

**Pull request closed without merging**